backend/openapi.yaml

```diff
@@ -1256,18 +1256,23 @@ paths:
                   type: string
                 recipient:
                   type: string
+                cc:
+                  type: array
+                  items:
+                    type: string
+                bcc:
+                  type: array
+                  items:
+                    type: string
                 from:
                   type: string
                 reply_to:
                   type: string
-                  nullable: true
```

Comment:
these are just optional (which is the default in OpenAPI), not nullable. (same as the ones below)